index.js

```diff
@@ -380,17 +382,28 @@
     };
   }
 
+  function typeX(name, test, _1, _2) {
+    var version = '0.7.0';  // updated programmatically
+    var url = 'https://github.com/sanctuary-js/sanctuary-def/tree/v' +
+              version + '#' + stripNamespace(name);
+    switch (arguments.length) {
```

Comment:
Great suggestion, Irakli! I was not at all happy with `typeX`. I've updated the pull request. :)